converters/fromZigbee.js

```diff
@@ -556,7 +557,10 @@ const converters = {
                 result.color_options = {execute_if_off: ((msg.data.options & 1<<0) > 0)};
             }
 
-            return result;
+            // handle color property sync
```

Comment:
I think it's still important to read xy+hs when doing a /get/color and color_temp when doing a /get/color_temp if color_sync=false. 

So something like:
color_sync=false -> same behavior as now + read color_mode so we at least color_mode will be updates with whatever the device is currently using
color_sync=true -> read color_mode, colorTemperature +  read for colorTemp, currentX, currentY, currentHue, currentSaturation, or do we go for one read with everything? I know xy+hs works fine in one read not sure we can push that to 6 attributes? I tested the Innr and it worked fine. I guess I could just test the tint, osram, ikea and hue too. If all of them support 6 attributes it's probably safe to do so.

Reply:
Hmm... something else I just thought about. To avoid UNSUPPORTED_ATTRIBUTE we need a way to check if the bulb supports colorTemperature or not. (Like we do for currentHue/currentSaturation)

I guess having unique convertGet's the following is easiest:
light_color_colortemp -> read all of them (* in one or two reads)
light_color -> read call of them except colorTemperature
light_colortemp -> read colorTemperature (I guess technically we should also read colorMode and currentX/currentY, given that scenes pre ZCLv7 would use xy for colorTemperature even if the bulb does not support xy 😞 )